README.md

```diff
@@ -14,17 +14,44 @@
 
 ## Documentation
 
-You can fine the preliminary documentation [here](./docs/README.md)
+1. [End User Getting Started Guide](./docs/getting-started.adoc)
+1. [Service Contributor Guide](./docs/contributing-guide.adoc)
+
+### List of SDKs
+
+AeroGear Services SDK consist of set of separate SDKs
+
+- [Core](./docs/core/README.adoc): Common base for all SDKs
+- [Metrics](./docs/metrics/README.adoc): Mobile application metrics SDK
+- [Auth](./docs/auth/README.adoc):  Mobile authentication SDK
+
+## Contributing
+
+See [General Contributing Guide](./CONTRIBUTING.md)
+
+## License 
+
+ See [LICENSE file](./LICENSE)
+
 
 ## Development
 
 If you would like to help develop AeroGear you can join our [developer's mailing list](https://groups.google.com/forum/#!forum/aerogear), join #aerogear on Freenode, or shout at us on Twitter @aerogears.
 
 Also takes some time and skim the [contributor guide](CONTRIBUTING.md)
 
+## Contributing
+
+See [General Contributing Guide](./CONTRIBUTING.md)
+
+## License 
```

Comment:
looks like this section and the above section are duplicated?

Reply:
Done